src/registry/registry.service.ts

```diff
@@ -94,6 +94,36 @@ export class RegistryService extends Service {
       .filter((v) => !!v);
   }
 
+  async getDevelopmentVaults(): Promise<VaultRegistryEntry[]> {
+    const developer = await this.registry.developer();
+    return this.getAuthorVaults(developer);
+  }
+
+  async getAuthorVaults(author: string): Promise<VaultRegistryEntry[]> {
+    if (!this.hasRegistry()) {
+      this.debug(
+        `${this.config.network} does not have a registry. No vaults are discoverable.`,
+      );
+      return [];
```

Comment:
not hooked up for now?

Reply:
this is empty only when the registry does not exist, it is not possible to look up vaults if a registry is not deployed on a network